Fronter/Source/Utils/LinuxFunctions.cpp

```diff
@@ -0,0 +1,17 @@
+#include "OSFunctions.h"
+#include "OSCompatibilityLayer.h"
+#include <codecvt>
+#include <filesystem>
+namespace fs = std::filesystem;
+
+std::string UTF16ToUTF8(const std::wstring& theString)
+{
+	std::u16string u16str(theString.begin(), theString.end());
+	std::wstring_convert<std::codecvt_utf8_utf16<char16_t>, char16_t> conversion;
+	return conversion.to_bytes(u16str);
+}
+
+std::optional<std::string> getSteamInstallPath(const std::string& steamID)
+{
+	return std::nullopt;
```

Comment:
As a future task, we can ask people to confirm the install locations mentioned here: https://askubuntu.com/questions/227502/where-are-steam-games-installed

Reply:
I'll confirm this myself as soon as I get to a linux computer. I think it's safe to assume Ubuntu is the distro of choice that we should target.